Library/Formula/emacs.rb

```diff
@@ -61,24 +66,28 @@ def install
     args = ["--prefix=#{prefix}",
             "--enable-locallisppath=#{HOMEBREW_PREFIX}/share/emacs/site-lisp",
             "--infodir=#{info}/emacs"]
+
```

Comment:
Inserted some empty lines between the deps blocks here, because emacs looks a little unreadable at the moment and spacing helps. If the empty lines are hated, let me know and I'll kill them.